documentation/markdown_and_code_dev.md

```diff
@@ -9,44 +9,67 @@ These include {term}`Jupyter Notebook`s, Linux {term}`command-line <terminal>` a
 
 A Jupyter Notebook is a feature-rich application supporting data exploration and code development.
 The notebook documents may be in either `.ipynb` or `.md` (Markdown) format.
-The Fornax Tutorial Notebooks are in `.md` format following the standard practice of the Scientific Python ecosystem.
+The Fornax Tutorial Notebooks are in `.md` format ([MyST](https://mystmd.org/) Markdown flavor) following the standard practice of the Scientific Python ecosystem.
 While many astronomers are familiar with the `.ipynb` format, we chose Markdown for its superior readability, ease of version control through diffing, simpler testing workflows, and smooth rendering to HTML.
 To read more about the differences between `.md` and `.ipynb`, see the [MyST Markdown documentation](https://mystmd.org/guide/md-vs-ipynb).
 
 ### JupyterLab Markdown Dependencies
 
 On the Fornax Science Console, all required dependencies for using Markdown in JupyterLab are pre-installed.
+
 To use Markdown notebooks in JupyterLab outside of the Fornax Science Console, ensure the following tools are installed:
 
 -   [`jupytext`](https://github.com/mwouts/jupytext) Python library
 -   [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst) JupyterLab extension
 
+(open-md-in-jupyterlab)=
 ### Opening `.md` Files in JupyterLab
 
-1.  Navigate to the `.md` file in the JupyterLab file browser.
-2.  Right-click the file, select **Open With**, and choose **Notebook** or **Jupytext Notebook**.
-3.  The Markdown file will open in an interactive notebook interface with cells and execution capabilities—just like a traditional `.ipynb` notebook.
+In the Fornax Science Console, you can simply double click on a `.md` file in the file browser.
+It will open in an interactive notebook interface with cells and execution capabilities — just like a traditional `.ipynb` notebook.
+If you need to save the notebook output, pair it with a `.ipynb` document before running it by selecting: `File → Jupytext → Pair Notebook with ipynb document`.
+
+If you are not in the Fornax Science Console then the default viewer for `.md` files might be different.
+In that case, right-click the file, select **Open With**, and choose **Notebook** or **Jupytext Notebook** to open it in the interactive notebook interface.
 
-### Converting `.md` Markdown Notebooks to `.py` Scripts
+### Converting To and From `.md` Format
 
-It is easy to convert from `.md` Markdown to a `.py` script, if that fits better into your workflow:
+:::{tip}
+It is not necessary to convert `.md` to `.ipynb` before using a notebook!
+Notebooks in `.md` format can be used in JupyterLab just like a traditional notebook.
+See {ref}`open-md-in-jupyterlab`.
+:::
+
+It is easy to convert notebooks between file formats like `.md`, `.ipynb`, and `.py` using [jupytext](https://jupytext.readthedocs.io/):
```

Comment:
Maybe this is too passive agressive, but I would emphasize it again that there is no reason to convert :)
```suggestion
While conversion is not at all necessary, if you prefer it is easy to convert notebooks between file formats like `.md`, `.ipynb`, and `.py` using [jupytext](https://jupytext.readthedocs.io/):
```

Reply:
the keep that in parens that conversions between md--> ipynb is not really necessary? (Most of the time I feel people just do that as a default first step because they are unaware of md and not because they need an ipynb only feature)

Reply:
It already says that again a few lines below here where the command is given. Three times in this same section is too many. If users really want to convert to .ipynb they can, it won't hurt anything.